sample_metadata/main.py

```diff
@@ -54,15 +55,15 @@ def sample_metadata(data: Dict[Literal['data'], str], unused_context: Any):
         'hail_version': hail_version,
         'batch_url': batch_url,
         'submitting_user': user,
-        'meta': metadata,  # any remaining keys
         'output_path': output_prefix,
     }
+    q = '&'.join([f'{k}={v}' for k, v in query_params.items()])
```

Comment:
I think probably better to use urllib here so that any escaping is handled. 

```python
from urllib.parse import urlencode

q = urlencode(query_params)
```

Reply:
Thanks for the suggestion, have applied 🙏